alica_engine/include/engine/PlanBase.h

```diff
@@ -88,11 +88,11 @@ class PlanBase
     TeamObserver& _teamObserver;
     TeamManager& _teamManager;
     const PlanRepository& _planRepository;
-    bool _stepEngine;
-    bool _stepCalled;
+    bool& _stepEngine;
+    bool& _stepCalled;
```

Comment:
I don't see anything in PlanBase.cpp using these, so I think you can take them out